src/xxd/dump.rs

```diff
@@ -239,7 +243,7 @@ where
 
 fn dump_line(data: &[u8], writer: &mut Write, output_settings: Config) {
     let output_line = OutputLine::new(data).format(output_settings);
-    writer.write_fmt(format_args!("{}\n", output_line));
+    writer.write_fmt(format_args!("{}\n", output_line)).unwrap();
```

Comment:
It would be nice to use a `?` here. However, the calling function is `dump_iterator` which returns a `Result<(), failure::Error>` losing all information that the writer conveys about an error. 

I suspect this really requires a refactor and then a `?` would indeed be better.

For the time being I left it at an `unwrap()`, but I agree, it shouldn't stay that way.

Reply:
Perfect thanks!